.github/workflows/test.yml

```diff
@@ -101,7 +101,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install tox tox-gh>=1.2
+          python -m pip install tox tox-gh>=1.2 -c constraints-dev.txt
```

Comment:
all jobs that read from constraints-dev.txt that also have `paths` section above should list the file in their `paths` filter, otherwise a change to the file won't trigger the job.